about/router/routes.js

```diff
@@ -0,0 +1,5 @@
+FlowRouter.route("/about", {
+  action: function (params) {
+    BlazeLayout.render("mainLayout", {about: "about"});
```

Comment:
If I use container-fluid, there will be paddings on both sides on the photo, and there are no padding on the mockup, therefore, I had to create a specific Template.dynamic for the about-page on the layout-page (similar to the landing page), to avoid container-fluid.